cdc/log.cc

```diff
@@ -1258,6 +1258,52 @@ class transformer final : public change_processor {
     // Takes and returns generated cdc log mutations and associated statistics about parts touched during transformer's lifetime.
     // The `transformer` object on which this method was called on should not be used anymore.
     std::tuple<std::vector<mutation>, stats::part_type_set> finish() && {
+        static const auto preimg_op_bytes  = _op_col.type->decompose(operation_native_type(operation::pre_image));
+        static const auto postimg_op_bytes = _op_col.type->decompose(operation_native_type(operation::post_image));
+
+        // Adjust or remove deltas according to `delta_mode` in `cdc_options`.
```

Comment:
I'm afraid I missed the motivation of this approach... Why add the deltas first as rows, and then remove them? Couldn't we just *not* add them?

Reply:
I asked @jul-stas  to make it this way. @kbr- is in the middle of refactoring of CDC code so it's better to keep this "no delta" option separated from the rest of the code. Later when @kbr- is finished we will integrate it back into the main code path.

Reply:
The refactoring is happening now. It's not in the future.